designer/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/EspItTest.scala

```diff
@@ -292,12 +294,52 @@ trait EspItTest extends LazyLogging with WithHsqlDbTesting with TestPermissions
         url += s"&isDeployed=$isDeployed"
       }
 
+      query.categories.foreach { categories =>
+        url += s"&categories=${categories.mkString(",")}"
+      }
+
+      query.processingTypes.foreach { processingTypes =>
+        url += s"&processingTypes=${processingTypes.mkString(",")}"
+      }
+
+      query.names.foreach { names =>
+        url += s"&names=${names.mkString(",")}"
+      }
+
       url
     }
 
   }
 
-  case class ProcessesQuery(categories: List[String], isSubprocess: Option[Boolean], isArchived: Option[Boolean], isDeployed: Option[Boolean])
+  case class ProcessesQuery(isSubprocess: Option[Boolean] = None,
```

Comment:
Do we need this class in tests? Why don't we just use FetchProcessesDetailsQuery? Or is it here for different reasons and it's better to keep it separate?

Reply:
I've switched to the case class used in directives.